src/components/gcds-alert/gcds-alert.tsx

```diff
@@ -0,0 +1,108 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { assignLanguage } from '../../utils/utils';
+
+@Component({
+  tag: 'gcds-alert',
+  styleUrl: 'gcds-alert.css',
+  shadow: true,
+})
+export class GcdsAlert {
+  @Element() el: HTMLElement;
+
+  private lang: string;
+
+  /**
+   * Defines the alert heading.
+   */
+  @Prop() alertHeading!: string;
+
+  /**
+   * Defines alert role.
+   */
+  @Prop() alertRole?: 'destructive' | 'info' | 'success' | 'warning' = 'info';
+
+  /**
+   * Defines if the alert's close button is displayed or not.
+   */
+  @Prop() hideCloseBtn?: boolean = false;
+
+  /**
+   * Defines the max width of the alert content.
+   */
+  @Prop() maxContentWidth?: 'fluid' | 'lg' | 'md' | 'sm' | 'xs' = 'lg';
+
+  /**
+   * Defines if the alert's position is fixed.
+   */
+  @Prop() positionFixed?: boolean = true;
+
+  /**
+   * Callback when the close button is clicked.
+   */
+  @Prop() onDismiss: () => void;
+
+
+  /**
+  * Events
+  */
+
+  if ( onDismiss ) {
+    onDismiss();
+  }
+
+  async componentWillLoad() {
+    // Define lang attribute
+    this.lang = assignLanguage(this.el);
+  }
+
+  render() {
+    const { alertHeading, alertRole, hideCloseBtn, lang, maxContentWidth, onDismiss, positionFixed } = this;
+
+    return (
+      <Host>
+        <div
+          class={`gcds-alert role-${alertRole} ${positionFixed ? 'is-fixed' : ''}`}
+          role="alert"
+          aria-label={
+            lang == 'en' ?
+              `This is ${ alertRole == 'info' ? 'an' : 'a'} ${alertRole} alert.`
```

Comment:
I really like this label but I am worried that when using the alert in `alert-role="destructive"` that the term destructive won't be clear to the user. It is consistent with our naming scheme but that's something a developer really only sees.

Doesn't mean it has to change, just want to flag it for when we do testing.

Reply:
That's a good point, I'm going to change it to 'critical' for the label instead. I will also add a note for myself to discuss with Amy if we should add any additional content for the destructive alert specifically, for example something like "Please take immediate reaction".